src/main/java/com/exasol/adapter/dialects/AbstractSqlDialect.java

```diff
@@ -143,15 +143,18 @@ public SchemaMetadata readSchemaMetadata(final List<String> tables) {
         return createRemoteMetadataReader().readRemoteSchemaMetadata(tables);
     }
 
-    @Override
-    public String getStringLiteral(final String value) {
+    public String quoteLiteralStringWithSingleQuote(final String value) {
```

Comment:
There is one caveat with this that worries me a little bit. There is an assumption here what the escape character is. In this case a two single quotes directly after each other. But that can differ between dialects again. Implementers might not think about this tough.